project.clj

```diff
@@ -20,4 +20,32 @@
               :fail-threshold 95}
   :ring {:handler api.handler/entry}
   :target-path "target/%s"
-  :profiles {:uberjar {:aot :all}})
+  :profiles {
```

Comment:
What's this?

Reply:
I set up a separate project profile for pact cause it has a **ton** of dependencies that I wouldn't want to be bundled with the final executable, they're just for contract testing.